lib/apiForSheets.js

```diff
@@ -188,6 +188,8 @@ module.exports.checkClaStatus = function(contextData) {
 
   // Authorize a client with the loaded credentials, then call the
   // Google Sheets API.
-  module.exports.authorize(
+  const oauthClient = module.exports.authorize(
     JSON.parse(clientSecret), module.exports.checkClaSheet);
+
+  module.exports.checkClaSheet(oauthClient);
 };
```

Comment:
Can you explain the purpose of this change i.e. why is it required for running the tests?

Reply:
Thanks for the explanation!